src/server/database/migrations/20180423095250_create-projects.js

```diff
@@ -0,0 +1,23 @@
+export const up = async knex => {
+  await knex.schema.createTable('projects', table => {
+    table
+      .increments('id')
+      .unsigned()
+      .primary()
+    table.string('name', 30).notNull()
+    table
+      .string('slug')
+      .unique()
+      .notNull()

```

Comment:
Slug doesn’t really have to be unique… it has to be unique for a specific user :/ I’m not sure how to deal with this

Reply:
I guess the name should be unique for a specific user.